README.md

```diff
@@ -377,6 +356,40 @@ it("should return transaction data and transaction receipt", async function () {
 
 For more usage examples, including tuple, array and struct support, as well as wallet support, check [sample-test.ts](https://github.com/Shard-Labs/starknet-hardhat-example/blob/master/test/sample-test.ts) of [starknet-hardhat-example](https://github.com/Shard-Labs/starknet-hardhat-example).
 
+### Devnet examples
+
+#### L1-L2 communication (Postman message exchange with Devnet)
+
+Exchanging messages between L1 ([Ganache](https://www.npmjs.com/package/ganache), [Hardhat node](https://hardhat.org/hardhat-network/#running-stand-alone-in-order-to-support-wallets-and-other-software), Ethereum testnet) and L2 (only supported for [starknet-devnet](https://github.com/Shard-Labs/starknet-devnet)) can be done using this plugin:
```

Comment:
Did you just move this section or are there any changes?

Reply:
Just moved it